tests/components/velux/test_config_flow.py

```diff
@@ -0,0 +1,132 @@
+"""Test the Velux config flow."""
+from __future__ import annotations
+
+from copy import deepcopy
+from typing import Any
+from unittest.mock import patch
+
+from pyvlx import PyVLXException
+import voluptuous as vol
+
+from homeassistant import data_entry_flow
+from homeassistant.components.velux import DOMAIN
+from homeassistant.config_entries import SOURCE_IMPORT, SOURCE_USER
+from homeassistant.const import CONF_HOST, CONF_PASSWORD
+from homeassistant.core import HomeAssistant
+import homeassistant.helpers.config_validation as cv
+
+DUMMY_DATA: dict[str, Any] = {
+    CONF_HOST: "127.0.0.1",
+    CONF_PASSWORD: "NotAStrongPassword",
+}
+
+PYVLX_CONNECT_FUNCTION_PATH = "pyvlx.PyVLX.connect"
+PYVLX_DISCONNECT_FUNCTION_PATH = "pyvlx.PyVLX.disconnect"
+
+
+def patch_async_setup_entry():
+    """Path the async_setup_entry function."""
+    return patch("homeassistant.components.velux.async_setup_entry", return_value=True)
```

Comment:
You should instead make this a fixture in a local `conftest.py`
https://github.com/home-assistant/core/blob/876776e2915c8c1d18b6bdfd82a0f25dc3d6b2ea/tests/components/abode/conftest.py#L12-L18

And use it automatically inside the tests here with pytestmark
https://github.com/home-assistant/core/blob/876776e2915c8c1d18b6bdfd82a0f25dc3d6b2ea/tests/components/abode/test_config_flow.py#L21